runners/katacoda/index.ts

```diff
@@ -186,17 +186,35 @@ export class Katacoda extends Runner {
         let fileDir = path.join(workspaceDir, runCommand.command.parameters[0]).replace(/\\/g, "/");
         let placeholder = runCommand.command.parameters[1].placeholder ? runCommand.command.parameters[1].placeholder : "";
         let dataTarget = runCommand.command.parameters[1].placeholder ? "insert" : "replace";
-
         let content = "";
+
+        if(runCommand.command.parameters[1].lineNumber)
+        {
+            let BGscript = "#!/bin/sh \n  \n sed 'lineNumber!#PLACEHOLDER#!' textdatei -i";
+            let number = parseInt(runCommand.command.parameters[1].lineNumber, 10);
+            let numberStr = (number == 1) ? (number.toString()+"i") : ((number-1).toString()+"a");
+            BGscript =BGscript.replace("lineNumber", numberStr);
```

Comment:
It would be better to use templates for the scripts. Then you just have to pass the parameters to the renderTemplates method and don't have to replace the content here